### ISSUE TITLE
gcc version warning in Makefile.PL

### ISSUE THREAD

**Issue by cjuerg** [perl5-dbi/DBD-Oracle#168]
Line 889-890 in Makefile.PL are now:
889: print "WARNING: Your GNU C compiler is very old. Please upgrade.\n"
890:    if ($Config{gccversion} and $Config{gccversion} =~ m/^(1|2\.[1-5])/);

Now regexp in 890 throws the "WARNING:..." with "gcc=10." and "gcc=20."
If 890 is changed to
890:        if ($Config{gccversion} and $Config{gccversion} =~ m/^(\b[1]\.\d|\b[2]\.\b[0-5]\.)/);

no "WARNING:..." will be thrown for any gcc version > 2.5

I tested it with the following code snippet:
for m in {1..20} ; do for s in {0..20} ; do for u in {0..20} ; do echo $m.$s.$u && /usr/bin/perl -e "my \$version='"$m.$s.$u"';print  'WARNING: Your GNU C compiler is very old. Please upgrade.' if (\$version =~ m/^(\b[1]\.\d|\b[2]\.\b[0-5]\.)/);" && echo ; done ; done ; done | less

**Comment by ozcoder** [comment]
I don't think it needs to be so complicated to just exclude **gcc** 1._X_ & 2.0, 2.1, 2.2, 2.3, 2.4 and 2.5. Although I would argue that good old 2.95 is way too old now too.

`m/^(1\.|2\.[1-5])/` would match the above versions. The `.` needed escaping.
You can see with
```
for m in {1..20} ; do \
for s in {0..20} ; do \
for u in {0..20} ; do \
echo "Version: "$m.$s.$u $(/usr/bin/perl -e "my \$version='"$m.$s.$u"';print  'WARNING: Your GNU C compiler is very old. Please upgrade.' if (\$version =~ m/^(1\.|2\.[1-5])/);") && echo ; \
done ; \
done ; \
done | grep -i warning
```

**Comment by djzort** [comment]
Pull requests are welcome, even for changes this trivial :)

**Comment by djzort** [comment]
I have released 1.95 https://metacpan.org/dist/DBD-Oracle
